File: CS593 C++/Calculator.cpp
#include<iostream>
#include <stdlib.h>

using namespace std;

class Calculator {
public:
	void calculate(float a, float b, char c);
};

void Calculator::calculate(float a, float b, char c) {
	if (c == '+')
		cout << a << " + " << b << " = " << (a + b) << endl;
	else if (c == '-')
		cout << a << " + " << b << " = " << (a - b) << endl;
	else if (c == '*')
		cout << a << " * " << b << " = " << (a * b) << endl;
	else if (c == '/')
		cout << a << " / " << b << " = " << (a / b) << endl;
	else
		cout << "Invalid input." << endl;
}

int main(int argc, char** argv) {
	float a = atof(argv[1]), b = atof(argv[3]);
	char c = argv[2][0];
	Calculator c1;
	c1.calculate(a, b, c);
	return (0);
}

File: CS594D OOP(Java)/Calculator.java
class Calculator {
	float calculate(float a, float b, char c) {
		if (c == '+')
			return (a + b);
		else if (c == '-')
			return (a - b);
		else if (c == '*')
			return (a * b);
		else if (c == '/')
			return (a / b);
		else
			return (-9999f);
	}
}

class Main {
	public static void main(String[] args) {
		float a = Float.parseFloat(args[0]), b = Float.parseFloat(args[2]);
		char c = args[1].charAt(0);

		Calculator c1 = new Calculator();

		if (c1.calculate(a, b, c) != -9999f)
			System.out.println(args[0] + " " + args[1] + " " + args[2] + " = " + c1.calculate(a, b, c));
		else
			System.out.print("Invalid input.");
	}
}